src/log_migration.rb

```diff
@@ -44,3 +45,12 @@
         end
     end
 end
+time = Time.utc 2015,8,26
+LogTools::Converter.register "converter for DepthMap missing the time field", time, Orocos.registry do
+    conversion "/base/samples/DepthMap","/base/samples/DepthMap" do |dst,src|
+        deep_cast(dst,src,:self)
+        if not src.timestamps.empty? then
```

Comment:
The time here matches the time of the log, not a hypothetical time tag attached to the type. People are, today, using Rock and generating DepthMap logs without a time field. They will do so for a while as e.g. they are using rock-15.05. So, the time tag allows us to avoid having to run all converters on all samples, but is not a silver bullet.